pkgs/by-name/ro/rollo-x1038/package.nix

```
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  cups,
+  lib,
+}:
+
+stdenv.mkDerivation {
+  name = "rollo-x1038";
+  version = "1.8.4";
+  src = fetchTarball {
```

Also prefer Nixpkgs' own fetchers
```suggestion
  src = fetchzip {
```